tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -531,13 +531,17 @@ public function removeAccessToSpace(
 		string $sharer,
 		string $shareType,
 		string $space,
-		?string $recipient = null
+		?string $recipient
```

Comment:
recipient is optional so let use null as default
```suggestion
		?string $recipient = null
```